powersimdata/data_access/data_access.py

```diff
@@ -180,21 +219,41 @@ def move_to(self, file_name, to_dir, change_name_to=None):
         :param str change_name_to: new name for file when copied to data store.
         """
         self._check_filename(file_name)
-        src = posixpath.join(server_setup.LOCAL_DIR, file_name)
+        src = os.path.join(server_setup.LOCAL_DIR, file_name)
         file_name = file_name if change_name_to is None else change_name_to
-        dest = posixpath.join(self.root, to_dir, file_name)
+        dest = os.path.join(self.root, to_dir, file_name)
         print(f"--> Moving file {src} to {dest}")
         self._check_file_exists(dest, should_exist=False)
-        self.copy(src, dest)
-        self.remove(src)
+        self.makedir(os.path.dirname(dest))
+        shutil.move(src, dest)
+
+    def makedir(self, full_path):
+        """Create path on local machine
+
+        :param str full_path: the path, excluding filename
+        """
+        os.makedirs(full_path, exist_ok=True)
+
+    @staticmethod
+    def _fapply(func, pattern):
+        files = [f for f in glob.glob(pattern) if os.path.isfile(f)]
+        for f in files:
+            func(f)
 
-    def makedir(self, relative_path):
-        """Create paths relative to the instance root
+    def copy(self, src, dest, recursive=False, update=False):
+        """Wrapper around cp command which creates dest path if needed
 
-        :param str relative_path: the path, without filename, relative to root
+        :param str src: path to original
+        :param str dest: destination path
+        :param bool recursive: create directories recursively
+        :param bool update: ignored
         """
-        target = os.path.join(self.root, relative_path)
-        os.makedirs(target, exist_ok=True)
+        self.makedir(dest)
```

Comment:
It seems that `copytree` will create `dest`. Should we move this line in the `else` then?